docs/docs/en/guides/features/cache/xcode-cache.md

```diff
@@ -0,0 +1,83 @@
+---
+{
+  "title": "Xcode Cache",
+  "titleTemplate": ":title · Cache · Features · Guides · Tuist",
+  "description": "Enable Apple's Compilation Artifact System (CAS) for your existing Xcode projects to share compilation artifacts across your team."
+}
+---
+# Xcode Cache {#xcode-cache}
+
+::: warning EXPERIMENTAL FEATURE
+This feature is currently experimental.
+:::
+
+Tuist provides support for the new Xcode caching capability, which allows teams to share compilation artifacts by leveraging the build system's caching capabilities.
+
+## Setup {#setup}
+
+::: warning REQUIREMENTS
+<!-- -->
+- A <LocalizedLink href="/guides/server/accounts-and-projects">Tuist account and project</LocalizedLink>
+- Xcode 26.0 or later
+<!-- -->
+:::
+
+If you don't already have a Tuist account and project, you can create one by running:
+
+```bash
+tuist init
+```
+
+Once you have a `Tuist.swift` file referencing your `fullHandle`, you can set up the caching for your project by running:
+
+```bash
+tuist setup cache
+```
+
+This command creates a LaunchAgent to run a local cache service on startup that the Swift build system uses to share compilation artifacts. This command needs to be run once in both your local and CI environments.
+
+To set up the cache on the CI, make sure you are <LocalizedLink href="/guides/integrations/continuous-integration#authentication">authenticated</LocalizedLink>.
+
+### Configure Xcode Build Settings {#configure-xcode-build-settings}
+
+Add the following build settings to your Xcode project:
+
+```
+COMPILATION_CACHE_ENABLE_CACHING = YES
```

Comment:
What about recommending to enable explicit modules? That might lead to greater parallelization.

Reply:
I'd still add a recommendation since it contributes to speeding up the build times through better parallelization.